male-seed-beetle/scripts/convert-with-core.py

```diff
@@ -1,7 +1,57 @@
 import polars as pl
+import re
+from pathlib import Path
+
+#Transform from tab sep to comma sep
+with open('../data-raw/data.csv', 'r') as tsv:
+
+    with open('../data-raw/data1.csv', 'w') as csv:
+
+        for line in tsv:
+
+            content = re.sub("\t", ",", line) 
+
+            csv.write(content)
+
+Path('../data-raw/data.csv').unlink()
+
+Path('../data-raw/data1.csv').rename('../data-raw/data.csv')
```

Comment:
This can actually be removed and keep both processed and the original, so you can always rerun this script without downloading the data again.